### ISSUE TITLE
syscall.Mmap under mount directory will got error: file resize error: invalid argument

### ISSUE THREAD

**Issue by Compasses** [boltdb/bolt#459]
Whether anyone have encountered such error. my GO projects host in windows, mount the directory in my virtual box Ubuntu actually . All my projects can run under these two platforms. But bolt DB such error as title.
if copy the run file on home directory it can work. i think it should be caused by the mount directory, but have no idea about this error. The mmap cannot work under mount directory file?

**Comment by arizvisa** [comment]
yes. a peer of mine had this same issue with the mmap call returning an invalid argument due to boltdb trying to mmap it's database off of the shared directory. if you copy it a real filesystem it should fix your issue.

**Comment by tv42** [comment]
Sounds like Virtualbox "shared folders" don't support mmap.
